aem-authoring-toolkit-plugin/src/test/com/exadel/aem/toolkit/test/widget/ColorFieldWidgetWithHtmlTag.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exadel.aem.toolkit.test.widget;
+
+import com.exadel.aem.toolkit.api.annotations.main.Dialog;
+import com.exadel.aem.toolkit.api.annotations.main.DialogLayout;
+import com.exadel.aem.toolkit.api.annotations.main.HtmlTag;
+
+@Dialog(
+        name = "test-component",
+        title = "test-component-dialog",
+        layout = DialogLayout.FIXED_COLUMNS
+)
+@HtmlTag(
```

Comment:
The only concern that naming HtmlTag is not so obvious in context of it's real purpose.
Basically even with aware of of AEM Decoration Tag feature I'm personally didn't get what is going on and what is the HtmlTag annotation.

Reply:
@ala-n , but isn't it less obvious for a user when annotation's name and XML node name are completely different?

Reply:
@ala-n I named this HtmlTag because we are developing AEM features and cq:htmlTag is node which is processing by AEM